src/client/South/SouthStatus.jsx

```diff
@@ -9,15 +9,15 @@ import { AlertContext } from '../context/AlertContext.jsx'
 const SouthStatus = () => {
   const [status, setStatus] = React.useState({})
   const { setAlert } = React.useContext(AlertContext)
-  const { dataSourceId } = useParams() // the dataSourceId passed in the url
+  const { id } = useParams() // the dataSource id passed in the url
 
   /**
    * Acquire the status
    * @returns {void}
    */
   const fetchStatus = () => {
     apis
-      .getSouthStatus(dataSourceId)
+      .getSouthStatus(id)
```

Comment:
Even if the name is retrieved in the status object, should the getSouthStatus ask for the name or the internal id ?

Reply:
Everything used to identify the connector (south or north) should be the id, and name should be used for display only.